frontend/src/app/person-edit/person-edit.component.spec.ts

```diff
@@ -133,10 +278,10 @@ describe('PersonEditComponent', () => {
     })));
 
     it('should have a title', () => {
-      const fixture = TestBed.createComponent(PersonEditComponent);
-      fixture.detectChanges();
+      const tester = new PersonEditTester(TestBed.createComponent(PersonEditComponent));
```

Comment:
Could that be simplified to: `new PersonEditTester(PersonEditComponent)`? Is there a reason to create and pass the fixture?